### PR TITLE
STRESS: Added 4 test cases

### PR DESCRIPTION
**Pull request by mbivolan**

### CODE REVIEW

File: Testscripts/Linux/Sysbench.sh [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+# Source utils.sh
+. utils.sh || {
+    LogErr "unable to source utils.sh!"
+    SetTestStateAborted
+    exit 0
+}
+
+LogMsg "Updating test case state to running"
+LogErr "Test Running"
```
Comment (chvalean) [inline Testscripts/Linux/Sysbench.sh]: please remove this

File: Testscripts/Windows/STRESS-BootLargeMemory.ps1 [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        $VMName,
+        $Ipv4,
+        $VMPort,
+        $VMUserName,
+        $VMPassword,
+        $RootDir
+    )
+	
+	$params = $testParams.Split(";")
+    foreach ($p in $params) {
```
Comment (chvalean) [inline Testscripts/Windows/STRESS-BootLargeMemory.ps1]: wrong indentation

File: Testscripts/Windows/STRESS-BootLargeMemory.ps1 [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        -command "cat /proc/meminfo | grep MemTotal | xargs | cut -d ' ' -f 2"
+    if ($? -ne "True") {
+        LogErr "Unable to send command to VM."
+        return "FAIL"
+    }
+    $memInfo = [math]::floor($memInfo/1024)
+
+    #Check if free binary and /proc/meminfo return the same value
+    if ($guestReadableMem -ne $memInfo) {
+        LogMsg "Warning: free and proc/meminfo return different values"
```
Comment (chvalean) [inline Testscripts/Windows/STRESS-BootLargeMemory.ps1]: LogWarn?

File: Testscripts/Windows/STRESS-EventID-18602-Regression.ps1 [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+        }
+         $tokens = $p.Trim().Split('=')
+         if ($tokens.Length -ne 2) {
+            LogMsg "Warn : test parameter '$p' is being ignored because it appears to be malformed"
+        }
+         if ($tokens[0].Trim() -eq "count") {
+            $count = $tokens[1].Trim()
+        }
+    }
+     if ($rootDir -eq $null) {
```
Comment (chvalean) [inline Testscripts/Windows/STRESS-EventID-18602-Regression.ps1]: is this still required?

File: Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1 [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+        if (-not $?) {
+            LogErr "VM became unresponsive after changing MTU on VM to $i on iteration $iteration "
+            return "FAIL"
+        }
+        $iteration++
+    }
+    LogMsg "Successfully changed MTU for $iteration times"
+
+    # Check for file
+
```
Comment (chvalean) [inline Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1]: empty line

File: Testscripts/Windows/Set-VM-Memory.ps1 [hunk 6]
```diff
@@ -6,59 +6,85 @@
...
+        }
+    }
+
+    if (-not $?) {
+        LogErr "Error: Unable to set ${VMMemory} of RAM for ${vmName}"
+        return $False
+    }
+
+    LogMsg "Success: Setting $VMMemory of RAM for $vmName updated successful"
+
```
Comment (chvalean) [inline Testscripts/Windows/Set-VM-Memory.ps1]: empty line

**Comment by chvalean** [comment]
@mbivolan please post test results for Ubuntu and CentOS, along with the confirmation that PSAnalyzer was ran.

File: Testscripts/Windows/STRESS-BootLargeMemory.ps1 [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+            $startupMem = $fields[1].Trim()
+        }
+    }
+
+    #Define peak available memory in case of a problem. This value is the maximum value that a VM is able to
+    #access in case of MTRR problem. If the guest cannot access more memory than this value the MTRR problem occurs.
+    $peakFaultMem = [int]67700
+
+    #Get VM available memory
+    $guestReadableMem = RunLinuxCmd -username "root" -password $VMPassword -ip $Ipv4 -port $VMPort `
```
Comment (juhlee-microsoft) [inline Testscripts/Windows/STRESS-BootLargeMemory.ps1]: Replace root with a variable.

File: Testscripts/Linux/Sysbench.sh [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+    if [ $? -ne 0 ]; then
+        LogErr "Cannot find cpu.log."
+        SetTestStateAborted
+    fi
+
+    RESULT_VALUE=$(echo ${PASS_VALUE_CPU} | head -c2)
+    if  [ $RESULT_VALUE -lt 15 ]; then
+        CPU_PASS=0
+        UpdateSummary "CPU Test passed."
+    fi
```
Comment (lizzha) [inline Testscripts/Linux/Sysbench.sh]: should also log when CPU test fail

**Comment by mbivolan** [comment]
PSAnalyzer was ran and all the relevant changes were implemented.

Test results: all 4 tests are passing on CentOS 7.5 and Ubuntu 18.04 (HyperV, Gen1)

**Pull request merged**